docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -135,6 +135,43 @@ Name::      The name of the connector. The name is used to identify a  connector
 API URL::   An optional PagerDuty event URL. Defaults to `https://events.pagerduty.com/v2/enqueue`. If you are using the <<action-settings, `xpack.actions.whitelistedHosts`>> setting, make sure the hostname is whitelisted.
 Integration Key::   A 32 character PagerDuty Integration Key for an integration on a service, also referred to as the routing key.
 
+[float]
+[[Preconfigured-pagerduty-configuration]]
+==== Preconfigured action type properties: 
+
+[source,text]
+--
+ id: 'my-pagerduty'
+ name: preconfigured-pagerduty-action-type
+ actionTypeId: .pagerduty
+ config:
+   apiUrl: https://test.host
+ secrets:
+   routingKey: testroutingkey
+--
+
+`config:` define action type specific to the configuration. The config object contains the following properties:
+
+[cols="3*<"]
+|=====
+
+|`apiUrl:`
+ | URL string
+ | correspond to *API URL*.
+
+|=====
+
+`secrets:` define action type sensitive configuration:
```

Comment:
Should "define action type ..." go in the second column?  Also remove colon after configuration.